src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -729,5 +729,9 @@
     <Compile Include="System\Xml\Core\LocalAppContextSwitches.cs" />
     <Compile Include="Misc\ThisAssembly.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <!-- TODO: Remove this reference once packages are updated -->
```

Comment:
I'm just going to use the existing issue.  These can all be removed once we get updated packages, which should be tonight or tomorrow morning.